cmd/controller-manager/app/controller_manager.go

```diff
@@ -360,9 +367,55 @@ func StartControllers(s *options.ControllerManagerServer,
 	glog.V(1).Info("Starting shared informers")
 	informerFactory.Start(stop)
 
+	if availableResources[podPresetGVR] {
+		// if PodPreset API is enabled, launch PodPreset admission control initializer
+		glog.Infof("PodPreset APIs are enabled, starting initializer for PodPresets")
+		// Build the informer factory for service-catalog resources
+		informerFactory := servicecataloginformers.NewSharedInformerFactory(
+			serviceCatalogClientBuilder.ClientOrDie("shared-informers"),
+			0,
+		)
+		// settings shared informers are v1alpha1 API level
+		settingsSharedInformers := informerFactory.Settings().V1alpha1()
+
+		podPresetController, err := podpreset.NewController(
+			coreClient,
+			recorder,
+			newPodSharedInformer(coreClient),
+			settingsSharedInformers.PodPresets(),
+		)
+		if err != nil {
+			glog.Errorf("error creating podpreset initializer : %v", err)
+			return err
+		}
+		// TODO(droot): check appropriate concurrency for initializer workers
```

Comment:
Is this a @jpeeler todo now?

Reply:
a better way is to use TODO(#issue number)